### PR TITLE
Alert on GCP when home directory has < 512GB available

### PR DESCRIPTION
**Pull request by yuvipanda**
This weekend too we had an outage caused by a full disk. In the past we have gotten lucky and caught it before it happened.

Here, we instead alert whenever the filestore has <512GB in it. This alert will be sent to a pagerduty channel I've configured to post on our slack.

I've applied this already to the shared cluster, and when merged, I'll apply this to all our GCP clusters.

I also checked the pricing - this shouldn't cost us anything, as it's an alert policy on a pre-existing metric.

Fixes https://github.com/2i2c-org/infrastructure/issues/3084

### CODE REVIEW

**Review by consideRatio — APPROVED** [review]
It took a while for me to understand how this setup worked, so I've pushed a commit with: inline comments, a file name update, a data structure naming update.

If my commit makes sense to you @yuvipanda I consider this ready to be merged!

File: terraform/gcp/secret/enc-pagerduty-service-key.secret.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+pagerduty:
+    disk_space: ENC[AES256_GCM,data:Gu7hX3j1aPEdFf0SUY8WH0JIc2T1tdRmdDfayGge84g=,iv:NbxmjQSoMCTSaRGXfGrZyGja0FYN6HxZ/BZcLVZBVYc=,tag:gRsi6qfuGWRK7jFT4NGcEw==,type:str]
```
[the remark this answers is not in the document]
Comment (consideRatio) [reply terraform/gcp/secret/enc-pagerduty-service-key.secret.yaml]: I ended up pushing a commit with my suggestions to reduce the toil for you @yuvipanda, if the change I made looks good to you go for a merge!

**Comment by yuvipanda** [comment]
Thanks @consideRatio. I've flattened out the secret config even more, added documentation inside the encrypted file (including link to where the key was from). The existing structure was to mirror what we have in `uptime-checks/`.

**Review by consideRatio — APPROVED** [review]

**Comment by yuvipanda** [comment]
I've applied this everywhere now.

**Pull request merged**